pkg/wheel/parse_test.go

```diff
@@ -0,0 +1,80 @@
+package wheel
+
+import (
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/xerrors"
+)
+
+func TestParse(t *testing.T) {
+	vectors := []struct {
+		file string // Test input file
+		want []types.Library
+	}{
+		{
+			file: "testdata/wheel_normal",
+			want: WheelNormal,
+		},
+		{
+			file: "testdata/wheel_many",
+			want: WheelMany,
+		},
+		{
+			file: "testdata/wheel_django",
+			want: WheelDjango,
+		},
+	}
+
+	for _, v := range vectors {
+		t.Run(path.Base(v.file), func(t *testing.T) {
+			f, err := os.Open(v.file)
+			require.NoError(t, err)
+			got, err := walk(f)
```

Comment:
So, `walk` is unnecessary here. We can simply pass a single file to `Parse`. I believe it would remove most of METADATA files under `testdata`.